src/Form.jsx

```diff
@@ -463,7 +466,12 @@ function Form({
                 });
             client.submitStep(featheryFields);
         }
-        client.registerEvent(activeStep.key, submitData ? 'complete' : 'skip');
+
+        client.registerEvent({
```

Comment:
this should be called after line 476 which calculates the next step

Reply:
My bad.. happened while rebasing I guess. Will update it in a bit.

Reply:
Moved `registerEvent` to 476